qa/docker/shared_examples/image_metadata.rb

```diff
@@ -13,34 +13,20 @@
     expect(@image.json['Architecture']).to have_correct_architecture
   end
 
-  %w(license org.label-schema.license org.opencontainers.image.licenses).each do |label|
+  %w(org.label-schema.license org.opencontainers.image.licenses).each do |label|
     it "should set the license label #{label} correctly" do
       expect(@labels[label]).to have_correct_license_label(flavor)
     end
   end
 
-  %w(name org.label-schema.name org.opencontainers.image.title).each do |label|
+  %w(org.label-schema.name org.opencontainers.image.title).each do |label|
     it "should set the name label #{label} correctly" do
       expect(@labels[label]).to eql "logstash"
     end
   end
 
-  %w(maintainer).each do |label|
-    it "should set the name label #{label} correctly" do
-      expect(@labels[label]).to eql "info@elastic.co"
-    end
-  end
-
-  %w(description summary).each do |label|
```

Comment:
would `org.opencontainers.image.description` be fitting here?

Reply:
yes, it would. I'll do a quick follow up,